### DISCUSSION TITLE
[@types/google-publisher-tag] and [@types/googletag] are the same?

### DISCUSSION THREAD

**Discussion by sangxxh** [DefinitelyTyped/DefinitelyTyped#62894] [Issues with a @types package]
Both libraries provide typing for `googletag`. It is confusing why there are 2 type libraries for the same thing. Should one be removed?

[Google official doc](https://developers.google.com/publisher-tag/guides/use-typescript) instruct to install `@types/google-publisher-tag` so that looks like the official type definition. It was added in #61562

[a comment by an automated account is not shown]

**Comment by jimper** [comment]
@types/google-publisher-tag are official typings from the GPT team, which were only released relatively recently. Prior to this there were (and still are) a number of community-led projects that provided typings for GPT. We leave it up to the owners of those projects whether they'd like to retire them or not.

You can see our reasoning for releasing official typings in [this blog post](https://ads-developers.googleblog.com/2022/10/google-publisher-tags-add-official.html).
